CHANGELOG.md

```diff
@@ -13,6 +13,24 @@
   - `register` and `register_from_file` on class `snowflake.snowpark.udtf.UDTFRegistration`
   - `register` and `register_from_file` on class `snowflake.snowpark.stored_procedure.StoredProcedureRegistration`
   - `udf`, `udtf` and `sproc` in `snowflake.snowpark.functions`
+- Support `Column` as an input argument to a stored procedure call
+- Support `table_type` in `save_as_table`. You can now choose from three different `table_type` options: `"temporary"`, `"temp"` and `"transient"`
+- Added the ability to only get a query ID after evaluating a `DataFrame` instead of fetching results
```

Comment:
Is this an open API on internal? If internal then we don't need to mention. @sfc-gh-jdu

Reply:
Yeah I wanted to comment this same. This is for internal usage only IIRC.

Reply:
Internal only